public/src/components/teacher/Results.tsx

```diff
@@ -140,7 +140,7 @@ export class Results extends React.Component<IResultsProps, IResultsState> {
             const previousStatus = selectedSubmission.status;
             selectedSubmission.status = status;
             // if the submission is for manual review, update submission score from reviews
-            if (currentSubmissionLink.assignment.getSkiptests()) {
+            if (currentSubmissionLink.assignment.getReviewers() > 0) {
```

Comment:
Also, since we seem to use `getReviewers() > 0` to determine whether or not the assignment has tests to be executed, we should instead create a helper function, or extend the assignment type with a method `hasTests()` or `isManual()`. This way we can more easily change these things in one place later, if necessary. See also PR #464.

Reply:
Alternatively, it could be added as an extension method on the SubmissionLink type, if that's the main use for such a method.